scripts/lcc_to_class_subclass.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+
+from __future__ import print_function
+
+import json
+import os
+import string
+
+import requests  # python3 -m pip install requests
+
+# from typing import Tuple
+
+here = os.path.dirname(os.path.abspath(__file__))
+
+# Load two Python dicts with Library of Congress Classification conversion data
+with open(os.path.join(here, "lc_classifiers_letters_only.json")) as in_file:
+    lcc_letters_only = json.load(in_file)
+with open(os.path.join(here, "lc_classifiers_letters_and_numbers.json")) as in_file:
+    lcc_letters_and_numbers = json.load(in_file)
+out_file = None
+
+
+def get_ol_book_info(olid="OL26617202M"):
+    """
+    def get_ol_book_info(olid: str = "OL26617202M") -> str:
+
+    >>> get_ol_book_info()  # doctest: +ELLIPSIS
+    {'olid:OL26617202M': ...
+    """
+    url = "https://openlibrary.org/api/books?jscmd=details&format=json&bibkeys=olid:"
```

Comment:
It would be easier to read/update if the query parameters were passed in a dict, as supported by requests.

Reply:
This is a WIP in Draft mode.  Those two utility functions are present to allow testers to experiment if they are more comfortable entering OL codes rather than LCC codes.  Given that we want get detailed data formatted as json from a fixed endpoint, these functions only need to modify a single parameter.  Dict-driven URLs are verbose and by no means mandatory for fixed endpoints such as APIs.